### ISSUE TITLE
extraneous input 'payable' expecting {'from', 'constant', 'internal', 'private', 'public', Identifier}

### ISSUE THREAD

**Issue by miguelmota** [protofire/solhint#77]
The new syntax `address payable` is not working 

```solidity
pragma solidity >0.4.99 <0.6.0;


contract Example {
    address payable public alice;

    constructor(address payable _alice) public {
        alice = _alice;
    }
}
```

```bash
$ solhint contracts/Example.sol
line 5:12 extraneous input 'payable' expecting {'from', 'constant', 'internal', 'private', 'public', Identifier}
line 7:24 mismatched input 'payable' expecting {',', ')'}
line 7:38 mismatched input ')' expecting '{'
line 8:22 extraneous input '=' expecting {'from', 'constant', 'internal', 'private', 'public', Identifier}
line 10:0 extraneous input '}' expecting {<EOF>, 'pragma', 'import', 'contract', 'interface', 'library'}

packages/channel-contracts/contracts/Example.sol
  1:17  warning  Compiler version must be fixed                compiler-fixed
  1:25  warning  Compiler version must be fixed                compiler-fixed
  7:25  warning  Explicitly mark visibility in function        func-visibility
  8:9   error    Expected indentation of 4 spaces but found 8  indent
  8:9   warning  Explicitly mark visibility of state           state-visibility
  9:5   error    Expected indentation of 0 spaces but found 4  indent

✖ 6 problems (2 errors, 4 warnings)
```
